index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Portfolio</title>
+	<link rel="stylesheet" href="style.css"/>
+	<link rel="preconnect" href="https://fonts.gstatic.com"/>
+	<meta name="viewport" content="width=device-width, initial-scale=1"/>
+	<link href="https://fonts.googleapis.com/css2?family=Nixie+One&display=swap" rel="stylesheet"/>
+</head>
+<body class="background">
+	<header>
+		<nav>
+			<a href="#home">home.</a>
+			<a href="#projects">projects.</a>
+			<a href="#about">about.</a>
+			<a href="#contact">contact.</a>
+		</nav>
```

Comment:
On the iphone 5/se (the smallest device I ever bother with) the nav menu is slightly cut off on the right hand side

![Screenshot 2021-03-11 at 09 54 54](https://user-images.githubusercontent.com/19924378/110769265-e3c07400-824f-11eb-9032-42e838196eb4.png)